pretty_gpx/drawing/poster_image_cache.py

```diff
@@ -174,9 +174,9 @@ def from_gpx_data(gpx_data: AugmentedGpxData,
         ele_scatter, ele_fill_poly, stats = get_elevation_drawings(layout=layout,
                                                                    h_pix=h, w_pix=w,
                                                                    list_ele=gpx_data.track.list_ele,
+                                                                   list_cum_d=gpx_data.track.list_cumul_d,
                                                                    passes_ids=gpx_data.passes_ids,
                                                                    huts_ids=gpx_data.hut_ids,
-                                                                   daily_dist_km=gpx_data.daily_dist_km,
```

Comment:
The field `daily_dist_km` from AugmentedGpxData is no longer used. You can remove it